examples/sms/chicken.c

```diff
@@ -34,17 +36,16 @@ int rand_speed() {
 }
 
 void main() { 
-    int x = 0; 
-    int y = 0; 
-    int  i, j; 
+    unsigned char x = 0; 
```

Comment:
Optimizations so that the chicken example app runs after adding the interrupt protection to the routines which access the VDP hardware.